packages/common/src/intl/locales/en/app.json

```diff
@@ -41,7 +41,7 @@
   "cmdk.create-inbound": "Create: New Inbound Shipment",
   "cmdk.goto-dashboard": "Go to: Dashboard",
   "cmdk.goto-customer-requisition": "Go to: Customer Requisition",
-  "cmdk.goto-supplier-requisition": "Go to: Supplier Requisition",
+  "cmdk.goto-internal-order": "Go to: Internal orders",
```

Comment:
we only have 'customer requisition' in `CommandK.tsx` - might be time to add a goto for `internal orders` too

Reply:
ah yes, thanks